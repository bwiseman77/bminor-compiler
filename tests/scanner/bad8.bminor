/* @ me bro for this comment
