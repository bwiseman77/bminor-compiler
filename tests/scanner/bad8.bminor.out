-------------------------------------------
           token type  text
-------------------------------------------
             DIVISON  /
      MULTIPLICATION  *
          SCAN ERROR  @
               IDENT  me
               IDENT  bro
                 FOR  for
               IDENT  this
               IDENT  comment
